tree_format/_text.py

```diff
@@ -20,66 +20,90 @@
     """
     replacement = u''.join([
-        NEWLINE,
+        options.NEWLINE,
         u'\n',
         prefix])
     return formatted_node.replace(u'\n', replacement)
 
 
-def _format_tree(node, format_node, get_children, prefix=u''):
+def _format_tree(node, format_node, get_children, options, prefix=u''):
     children = list(get_children(node))
-    next_prefix = u''.join([prefix, VERTICAL, u'   '])
+    next_prefix = u''.join([prefix, options.VERTICAL, u'   '])
     for child in children[:-1]:
         yield u''.join([prefix,
-                        FORK,
-                        HORIZONTAL,
-                        HORIZONTAL,
+                        options.FORK,
+                        options.HORIZONTAL,
+                        options.HORIZONTAL,
                         u' ',
                         _format_newlines(next_prefix,
-                                         format_node(child))])
+                                         format_node(child),
+                                         options)])
         for result in _format_tree(child,
                                    format_node,
                                    get_children,
+                                   options,
                                    next_prefix):
             yield result
     if children:
         last_prefix = u''.join([prefix, u'    '])
         yield u''.join([prefix,
-                        LAST,
-                        HORIZONTAL,
-                        HORIZONTAL,
+                        options.LAST,
+                        options.HORIZONTAL,
+                        options.HORIZONTAL,
                         u' ',
                         _format_newlines(last_prefix,
-                                         format_node(children[-1]))])
+                                         format_node(children[-1]),
+                                         options)])
         for result in _format_tree(children[-1],
                                    format_node,
                                    get_children,
+                                   options,
                                    last_prefix):
             yield result
 
 
-def format_tree(node, format_node, get_children):
+def format_tree(node, format_node, get_children, options=Options()):
     lines = itertools.chain(
         [format_node(node)],
-        _format_tree(node, format_node, get_children),
+        _format_tree(node, format_node, get_children, options),
         [u''],
     )
     return u'\n'.join(lines)
 
 
+def format_ascii_tree(tree, format_node, get_children):
+    """ Formats the tree using only ascii characters """
+    return format_tree(tree,
+                       format_node,
+                       get_children,
+                       Options(FORK=u'|',
+                               LAST=u'+',
+                               VERTICAL=u'|',
+                               HORIZONTAL=u'-',
+                               NEWLINE=u'\n'))
```

Comment:
Rather than this, I would suggest making a top-level `ASCII_OPTIONS` value and use that in here.